src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,216 @@
+        }
+        return fillDriversForCar(car);
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT cars.id, cars.model, cars.manufacturer_id, "
+                + "manufacturers.name, manufacturers.country "
+                + "FROM cars "
+                + "JOIN manufacturers "
+                + "ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars.id = ? AND cars.is_deleted = false;";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet carSet = statement.executeQuery();
+            if (carSet.next()) {
+                car = parseCarWithManufacturerFromResultSet(carSet);
+            }
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get car by id: " + id + " from database", ex);
+        }
+        if (car != null) {
+            car.setDrivers(getDriversForCar(id));
+        }
+        return Optional.ofNullable(car);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        List<Car> cars = new ArrayList<>();
+        String query = "SELECT cars.id, cars.model, cars.manufacturer_id, "
+                + "manufacturers.name, manufacturers.country "
+                + "FROM cars "
+                + "JOIN manufacturers "
+                + "ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars.is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            ResultSet carsSet = statement.executeQuery();
+            while (carsSet.next()) {
+                cars.add(parseCarWithManufacturerFromResultSet(carsSet));
+            }
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get all cars from database", ex);
+        }
+        for (Car car : cars) {
+            car.setDrivers(getDriversForCar(car.getId()));
+        }
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String query = "UPDATE cars "
+                + "SET model = ?, manufacturer_id = ? "
+                + "WHERE id = ? AND is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2, car.getManufacturer().getId());
+            statement.setLong(3, car.getId());
+            statement.executeUpdate();
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't update car: " + car, ex);
+        }
+        deleteCarDrivers(car.getId());
+        fillDriversForCar(car);
```

Comment:
```suggestion
        deleteCarDrivers(car.getId());
        fillCarDrivers(car);
```
use consistent naming